org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.raml

```diff
@@ -99,3 +99,37 @@ traits:
                 }
              }
     
+/subscribe:
+  description: |
+    This resource manages subscriptions to platoforms connected to this hub.
+  displayName: Subscribe
+    
+  post:
+    description: |
+       Verifies a subscription to a platform on this hub.
```

Comment:
Nope.  The hub subscribe should only be called by the server when the verification is received.  Adding this step to the hub controller keeps cloud services from having to instantiate (or even know about) the platform level translators.

---
In reply to: [92022538](https://github.com/openT2T/translators/pull/136#discussion_r92022538) [](ancestors = 92022538)